### PR TITLE
Updated terraform template

### PR DESCRIPTION
**Pull request by gbstan92**

### CODE REVIEW

File: infrastructure/variables.tf [hunk 1]
```diff
@@ -1,14 +1,14 @@
-variable "env" { 
-    description = "Name of the environment to deploy infrastructure"
-    type = string
+variable "env" {
+  description = "Name of the environment to deploy infrastructure"
+  type        = string
```
Comment (timja) [inline infrastructure/variables.tf:3]: string is the default not really worth specifying normally

**Review by timja — APPROVED** [review]

File: environments/sbox/sbox.tfvars [hunk 2]
```diff
@@ -0,0 +1,7 @@
+env = "sbox"
+product = "neuvector"
+location = "uksouth"
+key_vault = {
+  name = "dtssharedservicessboxkv"
```
Comment (timja) [inline environments/sbox/sbox.tfvars]: we shouldn't be adding vars per environment they can be determined from a pattern if we can avoid it, this uses the env flag in it

File: environments/sbox/sbox.tfvars [hunk 3]
```diff
@@ -0,0 +1,7 @@
+env = "sbox"
```
Comment (timja) [inline environments/sbox/sbox.tfvars]: this should be set on the command line, in the terraform template

File: environments/sbox/sbox.tfvars (hunk 2 above)
Comment (gbstan92) [reply environments/sbox/sbox.tfvars]: agree, this was just for testing purposes

File: environments/sbox/sbox.tfvars (hunk 3 above)
Comment (gbstan92) [reply environments/sbox/sbox.tfvars]: resolved

File: infrastructure/provider.tf [hunk 4]
```diff
@@ -0,0 +1,3 @@
+provider "azurerm" {
```
Comment (timja) [inline infrastructure/provider.tf:1]: I think you backend block defined as well  `backend "azurerm" {}`

full example:
https://github.com/hmcts/github-management/blob/main/terraform/providers.tf#L3

File: pipeline/azure-pipelines.yml [hunk 5]
```diff
@@ -17,7 +17,25 @@ variables:
   isMaster: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
   connectedServiceName: DCD-CFT-Sandbox
   keyVaultName: infra-vault-sandbox
-  secretsFilter: 'azure-devops-token'
+  secretsFilter: azure-devops-token
+  project: application
+
+parameters:
```
Comment (timja) [inline pipeline/azure-pipelines.yml:23]: can this be a variable we shouldn't have this showing up in the UI as a parameter

Comment (mokainos) [reply pipeline/azure-pipelines.yml:23]: sorry, i didn't catch your comments. What are we referring to here ?

Comment (timja) [reply pipeline/azure-pipelines.yml:23]: something like this:
```suggestion
variables:
```

parameters show up in the UI when you click run new

**Review by timja — APPROVED** [review]

**Pull request merged**